cmd/agent/agent.go

```diff
@@ -69,7 +70,7 @@ func loop(c *cli.Context) error {
 		)
 	}
 
-	zerolog.SetGlobalLevel(zerolog.WarnLevel)
+	zerolog.SetGlobalLevel(zerolog.InfoLevel)
 	if zerolog.GlobalLevel() <= zerolog.DebugLevel {
```

Comment:
shouldn't this be after line 76 ?

Reply:
Good point that's pretty strange 🤔 Line 72-75 should be in general behind 85?

Reply:
changed it